meilisearch-release-changelog/v0.29.0.md

```diff
@@ -0,0 +1,134 @@
+# v0.29.0 release changelogs
+
+<!-- The following line should NOT be put in the official release changelogs -->
+⚠️ Since this is a release candidate (RC), we do NOT recommend using this version in production. This is only for test purposes. Please, feel free to [report bugs](https://github.com/meilisearch/meilisearch/issues/new/choose) and [give feedback about the new feature introductions](https://github.com/meilisearch/product/discussions).
+
+In this release, we integrated a demanded feature: requiring all query terms to be present in search results. Some users have been stuck in the pre-0.21 versions because of this.
+We also improved the indexing speed
+
+🧰 Most of the integrations (SDKs, clients, libraries) you use will be compatible with this version in the 4 next hours following the release. Sometimes it can take longer, maybe 24h, depending on some issues we can encounter during the release.
```

Comment:
```suggestion
🧰 Most integrations (SDKs, clients, libraries) will be compatible with this version within four hours of the release. Sometimes it can take longer, maybe 24 hours, depending on the issues we can encounter during the release.
```